content/en/tools/archiving-service/archiving-service-index.md

```diff
@@ -1,419 +1,404 @@
+* `source.schema` - Vault schema name, defaults to vault database schema.
+* `target.schema` - Backup schema name, optional, indicates that a backup schema should be created.
+* `target.url` - Backup schema archive URL, required if a backup schema is used.
+* `target.user` - Backup schema archive database user, required if a backup schema is used.
+* `target.password` - Backup schema archive database password, required if a backup schema is used.
+
+Passwords can be obfuscated using Corda's Config Obfuscator tool.
+
+The following is a sample configuration file:
+
+```text
+generator: PostgresGenerator
+driver: "org.postgresql.Driver"
+
+target: {
+    url: "jdbc:postgresql:postgres"
+    user: "archive"
+    password: "<{HNtZpbrOGM6GhYA6foh5PCCBanUtaebCjauKL8ur9EE=:PolqmEJ7JOM+Sqj3ZNAE+Ew9bqG1wVE=}>"
+    schema: "archive"
+}
+```
+
+## Using the backup schema
+
+you can configure the archiving process to create a temporary snapshot image of the archivable transactions
+and attachments from your Corda vault on a backup schema within the same database. This snapshot can then be used
+to restore the vault should the database fail during the archiving operation.
+
+For the backup schema to work, the Corda vault schema and the archive schema must reside on the same database but be
+managed by different schema owners.
+
+The Archive Service uses a separate JPA entity manager factory to manage the
+archive schema and copy data from the Corda schema to the archive schema.
+
+To create a copy of the vault on the backup schema the backup schema owner must have
+`SELECT` rights to the vault schema.
+
+To restore the vault from a copy on the backup schema the backup schema owner must
+either have `INSERT` rights to the vault schema or the `restore-snapshot` command has to be executed with the
+`--record` option.
+
+The `--record` option allows the user to capture the SQL to a file so that it can be
+executed by a database administrator who has the sufficient rights.
+
+The following sections contain sample SQL statements needed for creating a backup schema user.
+In these examples the backup schema is called `archive` and the vault schema is `corda`.
+
+### Postgres
+
+The following DDL statements can be used to create a backup schema user `archive`:
+
+```
+create user archive password 'archive';
+create schema archive AUTHORIZATION archive;
+grant usage on schema corda to archive;
+grant select on all tables in schema corda to archive;
+grant insert on all tables in schema corda to archive;
+```
+
+### Oracle
+
+The following DDL statements can be used to create a backup schema user `archive`:
+
+```
+create user archive identified by archive2 DEFAULT TABLESPACE users QUOTA unlimited ON users;
+grant CONNECT, RESOURCE to archive;
+```
+
+Oracle does not have a single grant option to provide rights to all the tables in a schema.
+This means that rights have to be current on a per table basis.
```

Comment:
Not sure "current" is correct here?

Reply:
I didn't write this content so not sure either. Leaving as is.